elao.app/README.md

```diff
@@ -167,6 +167,10 @@ build@integration:
 lint.php-cs-fixer@integration:
 	mkdir -p report/junit
 	vendor/bin/php-cs-fixer fix --dry-run --diff --format=junit > report/junit/php-cs-fixer.xml
+	
+lint.phpstan@integration:
```

Comment:
Did'nt you say that a junit plugin was necessary ?

Reply:
i was wrong, they added this feature recently in the core

Reply:
Wow, that's a great news !